File: javascript/ex026.js
let frase = "Arara Azul".toLowerCase().trim();

console.log(`A letra A aparece ${frase.split('a').length - 1} vezes na frase`);
console.log(`A letra A aparece na posição ${frase.indexOf('a') + 1}`);
console.log(`A ultima letra A aparece na posição ${frase.lastIndexOf('a') + 1}`);

File: python3/ex026.py
frase = "Arara Azul".lower().strip()

print(F"A letra A aparece {frase.count('a')} vezes na frase")
print(F"A letra A aparece  na posição {frase.find('a')+1}")
print(F"A ultima letra A aparece na posição {frase.rfind('a')+1}")
